pennylane/operation.py

```diff
@@ -406,6 +407,9 @@ def _mod_and_round(x, mod_val):
     return str([id(d) if qml.math.is_abstract(d) else _mod_and_round(d, mod_val) for d in op.data])
 
 
+FlattenOutput = tuple[tuple[TensorLike, ...], tuple[Wires, tuple[tuple[str, Any], ...]]]
```

Comment:
I wasn't originally able to tell what this type means. Maybe this instead?
```suggestion
FlatPytree = tuple[tuple[TensorLike, ...], tuple[Wires, tuple[tuple[str, Any], ...]]]
```